Security.md

```diff
@@ -2,6 +2,6 @@
 
 ## Reporting Security Bugs
 
-Please report security bugs found in this plugin through the [Patchstack Vulnerability Disclosure Program](https://patchstack.com/database/vdp/yoast-comment-hacks). 
+Please report security bugs found in this plugin through the [Patchstack Vulnerability Disclosure Program](https://patchstack.com/database/vdp/comment-hacks). 
```

Comment:
Let's make a task to email the PatchStack team to update the VDP

Reply:
Added an item in https://github.com/Emilia-Capital/comment-hacks/issues/116